transaction/core/src/memo.rs

```diff
@@ -56,6 +57,8 @@ use serde::{Deserialize, Serialize};
 use sha2::Sha512;
 use zeroize::Zeroize;
 
+type Aes256Ctr = Ctr64BE<Aes256>;
```

Comment:
See https://docs.rs/aes/0.7.5/src/aes/autodetect.rs.html#147 for the original definition of `Aes256Ctr`